rectools/metrics/classification.py

```diff
@@ -430,15 +460,25 @@ def calc_classification_metrics(
         If unexpected metric is present in `metrics`.
     """
     k_map = defaultdict(list)
+    merged_debias = {}
     for name, metric in metrics.items():
         k_map[metric.k].append(name)
+        if not isinstance(metric, (ClassificationMetric, SimpleClassificationMetric)):
```

Comment:
why do we need this check?

Reply:
This is the [branch in the condition](https://github.com/MobileTeleSystems/RecTools/blob/main/rectools/metrics/classification.py#L449)

Reply:
Yes, but it's not necessary, added just in case.  

However, double-checking here is really too much. Let's drop at least the last branch in the 2nd check

Reply:
Is it correct now?

Reply:
yes